src/plugins/intel_gpu/src/graph/include/activation_inst.h

```diff
@@ -24,6 +24,7 @@ struct typed_program_node<activation> : public typed_program_node_base<activatio
     using parent::parent;
 
     program_node& input() const { return *get_dependency(0).first; }
+    int32_t input_idx() const { return get_dependency(0).second; }
```

Comment:
Ummm it sounds like the index of input within this node, but it is pointing the output idx of the previous node for the corresponding input..... Could we come up with better naming for this method to be more intuitive?
For example, input_dep_idx, etc... thought its still blury.. :( (I am not good at naming though)

Reply:
How about "output_idx_of_input_node()" or "input_node_output_idx()"?

Reply:
hmmm how about input_dep_idx_to_prev_node ?

Reply:
I've changed to 'input_dep_idx'.